allauth/account/migrations/0001_initial.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import models, migrations
 import django.utils.timezone
 from django.conf import settings
+from django.db import migrations, models
+
```

Comment:
remove useless lines

Reply:
@yasinByasin94 , could you explain what do you mean under "useless lines"?
If you check the [whole file](https://github.com/pennersr/django-allauth/blob/13a733edd328b75619799bf09de5b7c4310ad54a/allauth/account/migrations/0001_initial.py) you will see that these imports are used.

Reply:
@DmytroLitvinov he probably meant that you added an empty line underneath imports

Reply:
@sicin, this is how `isort` formatted it.